widgets/conf/SiteDefs.pm

```diff
@@ -30,6 +30,8 @@ sub update_conf {
   $SiteDefs::REACTOME_JS_LIBRARY = 'https://www.reactome.org/DiagramJs/diagram/diagram.nocache.js';
 
   $SiteDefs::Pathway = 1;
+
+  $SiteDefs::PDB_EBI_URL = 'http://www.ebi.ac.uk/pdbe/pdb-component-library';
```

Comment:
```suggestion
  $SiteDefs::PDB_EBI_URL = '//www.ebi.ac.uk/pdbe/pdb-component-library';
```

Because of http and https protocol, its better to use // which will take the protocol by default. can you try the above suggestion and see if it works

Reply:
Should I do the same for the GXA URLs as well for consistency ?

Reply:
Yes please

Reply:
Oh no, sorry dont do the GXA one because i think there was a problem with it.